### DISCUSSION TITLE
Can not save/publish project with error "Updating the listing failed"

### DISCUSSION THREAD

**Discussion by saltict** [WalletConnect/walletconnect-monorepo#2980] [Developer Support] [answered]
Dear Support Team,

I hope this message finds you well. I am reaching out to report an issue I encountered while attempting to save and publish our Project as an injected wallet on https://cloud.walletconnect.com/. Unfortunately, I received the following error message during the process: "Updating the listing failed: 'invalid input syntax for type uuid: "undefined"'."

Despite my efforts to resolve the problem, such as reloading the page and ensuring that all required values are properly filled, I have been unable to make any progress. The error persists, preventing me from successfully completing the saving and publishing of our Project.

Could you kindly investigate this matter and provide assistance to help resolve the issue? I would greatly appreciate any guidance or instructions you can offer to help me overcome this obstacle and proceed with the desired task.

Thank you for your attention and prompt assistance in this matter. I look forward to hearing back from you soon.

Best regards.

**Comment by Cali93** [comment]
Hi @saltict,

What is your project id ? Can you try to log out and log in again ?

**Comment by saltict** [comment]
Hi @Cali93,

I have tried to clear data, but it doesn't work. If I changing the wallet type to "mobile wallet".I can save my data. 

After changing it back to "injected wallet" add new provider flag it returns another error **Updating the listing failed: "new row violates row-level security policy for table \"submission_injected_ids\""`**

My project id: 6da34c0b48164d27681924dd9a46d6be

**Comment by saltict** [comment, marked as answer]
Hi @Cali93,

The found that the problem was because of something was wrong in the validated injected wallet.

Tip & trick to fix this: Save all data as a mobile wallet and turn it back to injected wallet later.

**Reply by Cali93** [reply]
Glad to hear you managed to unblock yourself, thanks for your report. 
It helped us to spot a bug that will be fixed by end of day.
